bin/GameIndex.dbf

```diff
@@ -24867,6 +24935,11 @@ Region = NTSC-J
 Serial = SLPM-66008
 Name   = Musashiden II - Blademaster
 Region = NTSC-J
+[patches = F0E90890]
+	comment=Patch by Prafull
+	// Gets rid of some busy effects that cause the screen to flash rapidly, should be ok to pass unplayable areas.
```

Comment:
The first part should be `Fixes flashing screen issues.`
But it doesn't explain what unplayable areas means.

Reply:
That patch is only work in progress, I won't suggest adding it to the database anyway. Unplayable areas meant the places where it's impossible to see anything due to massive flashing screen issues.